tests/http/test_19_shutdown.py

```diff
@@ -74,6 +67,8 @@ def test_19_01_check_tcp_rst(self, env: Env, httpd, proto):
     @pytest.mark.skipif(condition=not Env.tcpdump(), reason="tcpdump not available")
     @pytest.mark.parametrize("proto", ['http/1.1', 'h2'])
     def test_19_02_check_shutdown(self, env: Env, httpd, proto):
+        if env.parallel_testing:
+            pytest.skip('does not work in parallel testing')
```

Comment:
Is there any way to force these tests to be run sequentially even when running in parallel mode? It's a shame to have to skip tests in this case. Maybe have each such test hold a global lock so all others are forced to wait? I didn't see anything in the xdist docs to help with this but it seems conceptually doable. Is there at least one CI job still with parallel disabled so these will run somewhere?

Reply:
I found a way to run these tests in parallel. I restrict the `tcpdump` checks to the ports the client is talking to. So there is no interference and remove the `env.parallel_testing` checks.